packages/app/src/app/pages/Dashboard/Content/routes/Settings/NewTeam.tsx

```diff
@@ -1,101 +1 @@
-import { dashboard as dashboardUrls } from '@codesandbox/common/lib/utils/url-generator';
-import css from '@styled-system/css';
-import history from 'app/utils/history';
-import { Card } from './components';
-
-export const NewTeam = () => {
-  const {
-    actions: { dashboard },
-  } = useOvermind();
-  const [loading, setLoading] = useState(false);
-
-  const onSubmit = async event => {
-    event.preventDefault();
-    const teamName = event.target.name.value;
-    if (teamName && teamName.trim()) {
-      event.target.name.setCustomValidity('');
-      setLoading(true);
-      try {
-        await dashboard.createTeam({
-          teamName,
-          pilot: location.search.includes('pilot'),
-        });
-        setLoading(false);
-        history.push(dashboardUrls.teamInvite());
-      } catch {
-        setLoading(false);
-      }
-    }
-  };
-
-  const handleInput = e => {
-    const { value } = e.target;
-    if (value && value.trim()) {
-      e.target.setCustomValidity('');
-    } else {
-      e.target.setCustomValidity('Workspace name is required.');
-    }
-  };
-
-  return (
-    <>
-      <Helmet>
-        <title>Create Workspace - CodeSandbox</title>
-      </Helmet>
-      <Element
-        css={css({
-          height: 'calc(100vh - 140px)',
-          overflowY: 'scroll',
-        })}
-      >
-        <Stack justify="center" align="center" css={css({ height: '100%' })}>
-          <Card
-            css={css({
-              maxWidth: 528,
-              height: 'auto',
-              paddingTop: 10,
-              paddingBottom: '68px',
-              paddingX: [6, '96px', '96px'],
-            })}
-          >
-            <Stack direction="vertical" gap={7}>
-              <Stack direction="vertical" gap={4}>
-                <Text size={6} weight="bold" align="center">
-                  Create a workspace
-                </Text>
-                <Text size={3} variant="muted" align="center">
-                  Collaborate on code with friends or co-workers. Manage and
-                  work on sandboxes together.
-                </Text>
-              </Stack>
-
-              <Stack as="form" onSubmit={onSubmit} direction="vertical" gap={2}>
-                <Input
-                  name="name"
-                  type="text"
-                  placeholder="Workspace name"
-                  autoFocus
-                  required
-                  onChange={handleInput}
-                  css={css({ height: 8 })}
-                />
-                <Button
-                  loading={loading}
-                  disabled={loading}
-                  type="submit"
-                  css={css({ height: 8, fontSize: 3 })}
-                >
-                  Create Workspace
-                </Button>
-              </Stack>
-            </Stack>
-          </Card>
-        </Stack>
-      </Element>
-    </>
-  );
-};
+export { NewTeam } from 'app/pages/common/NewTeam';
```

Comment:
Abstracting NewTeam into app/pages/common because both Dashboard and Pro use it now